pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go

```diff
@@ -20,13 +20,13 @@ import (
 )
 
 const (
-	// DefaultZkContainerRepository is the default docker repo for the zookeeper
-	// container
-	DefaultZkContainerRepository = "pravega/zookeeper"
+	// Default images
+	//DefaultZkContainerRepository = "pravega/zookeeper"
+	//DefaultZkContainerVersion = "0.2.8"
 
-	// DefaultZkContainerVersion is the default tag used for for the zookeeper
-	// container
-	DefaultZkContainerVersion = "0.2.8"
+	// Image built by Adobe ad-cloud
+	DefaultZkContainerRepository = "cr.k8s.tubemogul.info/jinhuan/zookeeper"
```

Comment:
I suggest we keep the defaults here from
Pravega upstream and use your CR to override this if needed. This way - submitting a PR upstream would be easier

Reply:
Sounds good. Updated it.